### DISCUSSION TITLE
Junit output type

### DISCUSSION THREAD

**Discussion by alexnot95** [lyndsey-ferguson/fastlane-plugin-test_center#350] [General]
Hi @lyndsey-ferguson I've added ```output_types: "html"``` parameter to my multi_scan lane to not face this issue - https://github.com/lyndsey-ferguson/fastlane-plugin-test_center/issues/339

but during the run on CI I see this message in console log: ```[worker 1] [14:17:19]: Scan output types missing 'junit', adding it```

Are there any possibility to avoid adding junit output type? Thanks

**Comment by lyndsey-ferguson** [comment]
Hi @alexnot95!

No, there isn't a way to avoid adding junit right now as multi-scan uses the report to know which tests to retry (or not retry). That is one reason why we have the feature request to switch to using xcresult files: https://github.com/lyndsey-ferguson/fastlane-plugin-test_center/issues/292.

I should be picking up more time to look into test center issues soon.

Lyndsey

**Reply by lyndsey-ferguson** [reply]
Keep on eye on that issue, I'm starting to plan it out!
